src/univalue/lib/univalue_get.cpp

```diff
@@ -58,6 +58,22 @@ bool ParseInt64(const std::string& str, int64_t *out)
         n <= std::numeric_limits<int64_t>::max();
 }
 
+
+bool ParseUint64(const std::string& str, uint64_t *out)
+{
+    if (!ParsePrechecks(str))
+        return false;
+    char *endp = NULL;
```

Comment:
nullptr here

Reply:
ill update all of the parse functions for this